src/pages/Team/Team.js

```diff
@@ -1,7 +1,34 @@
 import React from 'react';
+import constTeam from './ConstTeam';
 
 function Team() {
-  return <h1 className="mx-auto text-center text-7xl">Team</h1>;
+  return (
+    <div >
```

Comment:
suggest to add the following to line 6:
className="mx-auto lg:max-w-7xl px-10 py-10"